tool/tsh/tsh.go

```diff
@@ -953,7 +958,12 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 	// Connect to the span exporter and initialize the trace provider only if
 	// the --trace flag was set.
 	if cf.SampleTraces {
-		provider, err := newTraceProvider(&cf, command, []string{login.FullCommand()})
+		// login only needs to be whitelisted if forwarding to auth
+		var whitelist []string
+		if cf.TraceExporter == "" {
```

Comment:
Should we make the `--trace` enabled by default when the `trace-exporter` flag was set ? 
Right now a user needs to provide both flags `--trace --trace-exporter`  but  not handle the `--trace-exporter` with trace flag by default ?